chariott-proto/src/lib.rs

```diff
@@ -10,6 +10,12 @@ mod chariott {
     pub mod common {
         pub mod v1 {
             tonic::include_proto!("chariott.common.v1");
+            pub use fulfillment::Fulfillment as FulfillmentEnum;
```

Comment:
Didn't we say, these will be exposed directly under `chariott_proto` as opposed to `chariott_proto::common`?

Reply:
We could, but we would still need to re-export the original common module as it's used by https://github.com/eclipse/chariott/blob/main/examples/common/src/examples/proto.rs. There might be another way to solve this (recompiling the common proto), but not sure it's worth the effort at this point.